src/pages/index.js

```diff
@@ -84,26 +104,67 @@ const Index = () => {
             alignItems: 'stretch',
           }}
         >
+          <Img fluid={hero} alt='TJCSC at Lockheed Martin CYBERQUEST 2019'
+            sx={{
+              borderRadius: 4,
+              mb: 1,
+            }}
+          />
+          TJCSC at Lockheed Martin CYBERQUEST 2019
+        </Flex>
+      </Flex>
+      <Flex id='about'
+        sx={{
+          bg: 'altBackground',
+          '& > *': {
+            flex: '1 1 0',
+          },
+        }}
+      >
+        <Flex
+          sx={{
+            flexDirection: 'column',
+            justifyContent: 'center',
+            alignItems: 'center',
+          }}
+        >
+          <Heading
+            as='h1'
+            sx={{
+              m: [4, 5, 6],
+              fontSize: [5, 6, 7],
+            }}
+          >
+            The Club
+          </Heading>
+        </Flex>
+        <Img fluid={{ ...club, aspectRatio: 1.778 }} alt='TJ Computer Security Club Meeting, October 2016'
+          sx={{
+            display: ['none', null, 'block'],
+            mb: 1,
+          }}
+        />
+      </Flex>
+      <Flex
+        sx={{
+          bg: 'lightBackground',
+          flexDirection: ['column', null, 'row'],
+          '& > *': {
+            flex: '1 1 0',
+            justifyContent: 'space-between',
+            p: [4, 5, 6],
```

Comment:
Seems to be cram space, maybe something like `p: [4, null, 5],` would work better